.github/workflows/update.yaml

```diff
@@ -0,0 +1,49 @@
+name: Update
+
+# Enable while working on this file
+on:
+  push:
+    paths: 
+      - "update.yaml"
+
+# Enable for daily auto-updates
+# on:
+#     schedule:
+#     - cron: '07 8 * * *' # 08:07 UTC is 00:07 PST
+
+env:
+  ZONEDBOT_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This should be a separate and unique token, specifically for ZoneDBot, not `GITHUB_TOKEN`.

Reply:
I'm curious why, e.g. is there a different permission it needs?

https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token

Reply:
To prevent a cascade of events, pull requests or pushes created with the default `GITHUB_TOKEN` will not trigger further GitHub Actions.

You have to create and use your own token to create pull requests that you want to trigger tests (and in our case, auto-merge). This is why we use them in alta/p1.

Reply:
@case dusting this off. Did you add a custom `ZONEDBOT_GITHUB_TOKEN` secret?